documentation/sphinx/source/api-python.rst

```diff
@@ -792,6 +792,10 @@ Most applications should use the read version that FoundationDB determines autom
 
     |infrequent| |transaction-get-committed-version-blurb|
 
+.. method :: Transaction.get_approximate_size()
+
+    |infrequent| |transaction-get-approximate-size-blurb|
+
```

Comment:
I think this probably makes more sense in the section below for miscellaneous functions. Also, we probably don't need to tag it with `infrequent`, as that was meant to discourage people from manipulating versions unnecessarily.

Lastly, I think we should make sure to include in the blurb that this function is returning a future. Python and Ruby have some replacements that you could use for this purpose, like `|future-object|`.

The same comments apply to Ruby.

Reply:
Done.